### DISCUSSION TITLE
Is C#10 releasing with .NET 6.0?

### DISCUSSION THREAD

**Discussion by zmeghji** [dotnet/csharplang#4780] [Q&A] [answered]
Is it known when C#10 will be released and whether it will be going out with .NET 6.0?

**Comment by jnm2** [comment, marked as answer]
It will be released with .NET 6. General pattern: https://github.com/dotnet/csharplang/discussions/2518#discussioncomment-112588

Announcement at Microsoft Build yesterday reinforcing this: https://mybuild.microsoft.com/sessions/70d379f4-1173-4941-b389-8796152ec7b8 (video timestamp 0:07:52):

![image](https://user-images.githubusercontent.com/8040367/119676398-d2791380-be0b-11eb-87aa-b32cd8827233.png)

The release dates was just announced for .NET 6 also: https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-4/

> Speaking of the final release, we now have a date! Book off November 9-11 for .NET Conf 2021. We’ll launch .NET 6 on the 9th with many in-depth talks and demos that tell you everything you want to know about .NET 6.

**Reply by zmeghji** [reply]
Thanks @jnm2 !
